packages/mongo-nav/src/org-nav/GovBanner.tsx

```diff
@@ -0,0 +1,149 @@
+import React from 'react';
+import GovBuildingIcon from '@leafygreen-ui/icon/dist/GovBuilding';
+import Tooltip from '@leafygreen-ui/tooltip';
+import { css, cx } from '@leafygreen-ui/emotion';
+import { uiColors } from '@leafygreen-ui/palette';
+import { Body } from '@leafygreen-ui/typography';
+
+const desktopContainer = css`
+  margin-bottom: 10px;
+  position: relative;
+  z-index: 2;
+  display: flex;
+  flex-grow: 1;
+  height: 5px;
+  background-color: #edf5fa;
+  align-items: flex-start;
+  justify-content: center;
+`;
+
+const tabletContainer = css`
+  position: relative;
+  z-index: 2;
+  display: flex;
+  flex-grow: 1;
+  align-items: flex-start;
+  justify-content: center;
+  height: 0;
+`;
+
+const mobileContainer = css`
+  width: 30px;
+  height: 30px;
+  background-color: #edf5fa;
+  margin-left: 12px;
+  border-radius: 7px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  cursor: pointer;
+`;
+
+const sharedBannerContainerStyles = css`
+  background-color: #edf5fa;
+  display: flex;
+  align-items: center;
```

Comment:
Looks like we are setting this to `flex-start` for both desktop and mobile. We can probably remove those values and set that value here. Same for `display: flex` and `justify-content`.

Reply:
Shared Banner container only applies to desktop/tablet

Reply:
I understand – my point still stands I think.

Reply:
These are the styles being appended to sharedBannerContainer

```
const sharedBannerContainerStyles = css`
  background-color: #edf5fa;
  display: flex;
  align-items: center;
  justify-content: center;
  cursor: pointer;
  position: relative;
`;

const desktopTooltipContainer = css`
  height: 34px;
  width: 205px;
  border-radius: 0 0 17px 17px;
`;

const tabletTooltipContainer = css`
  height: 20px;
  width: 200px;
  border-radius: 0 0 15px 15px;
`;```

Reply:
Ahh, I was looking at `tabletContainer` and `desktopContainer`

Reply:
sorry the naming conventions i used aren't great... 🤦